app/src/androidTest/java/com/simplemobiletools/calculator/MainActivityTest.java

```diff
@@ -21,7 +21,7 @@
     @Rule public final ActivityTestRule<MainActivity> activity = new ActivityTestRule<>(MainActivity.class);
 
     @Test
-    public void addDigits() {
+    public void addDigitsTest() {
```

Comment:
*Style comment*. The current naming convention for tests are descriptive instead of prescriptive, ie. this is an `addDigitsTest()`, instead of, do `testAddDigits()`. Furthermore, the names could also include the expected result, ie. `testAddDigitsShouldDisplayDigits()`. Thoughts on changing this convention?

Reply:
You raise a valid point should be something to look into further on.